modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/index/contributor/ObjectEntryModelDocumentContributor.java

```diff
@@ -280,9 +280,36 @@ private void _contribute(Document document, ObjectEntry objectEntry)
 		StringBundler sb = new StringBundler(objectFields.size() * 4);
 
 		for (ObjectField objectField : objectFields) {
-			_contribute(
-				document, fieldArray, objectDefinition, objectEntry,
-				objectField, sb, values);
+			if (objectField.isLocalized()) {
+				Map<String, String> localizedValues =
+					(Map<String, String>)values.get(
+						objectField.getI18nObjectFieldName());
+
+				if (localizedValues.isEmpty()) {
+					continue;
+				}
+
+				for (Map.Entry<String, String> localeMap :
+						localizedValues.entrySet()) {
+
+					String locale = LocaleUtil.fromLanguageId(
+						localeMap.getKey(), true, false
+					).toString();
+
+					String fieldName = objectField.getName();
```

Comment:
Hey @JhossephG what is the reason why we are not append the language id in the field name to follow the standard of other assets?